zulip_bots/zulip_bots/bots/define/define.py

```diff
@@ -17,19 +19,19 @@ class DefineHandler(object):
     PHRASE_ERROR_MESSAGE = 'Definitions for phrases are not available.'
     SYMBOLS_PRESENT_ERROR_MESSAGE = 'Definitions of words with symbols are not possible.'
 
-    def usage(self):
+    def usage(self) -> str:
         return '''
             This plugin will allow users to define a word. Users should preface
             messages with @mention-bot.
             '''
 
-    def handle_message(self, message, bot_handler):
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
         original_content = message['content'].strip()
         bot_response = self.get_bot_define_response(original_content)
 
         bot_handler.send_reply(message, bot_response)
 
-    def get_bot_define_response(self, original_content):
+    def get_bot_define_response(self, original_content: Any) -> str:
```

Comment:
Can you see how `original_content` could be typed more specifically, based on how it's used in the function, or the values that are passed in?